data/ab23.json

```diff
@@ -0,0 +1,134 @@
+{
+  "id": 278,
+  "name": "AB:23",
+  "tagline": "Bourbon Barrel-Aged Barley Wine.",
+  "first_brewed": "2017",
+  "description": "Our Abstrakt series showcases one-off beers, brewed or blended to the peak of our abilities. Each release is unique and never to be repeated.",
+  "image_url": "null",
+  "abv": 11.5,
+  "ibu": 80,
+  "target_fg": 1020,
+  "target_og": 1110,
+  "ebc": 80,
+  "srm": 41,
+  "ph": 4.1,
+  "attenuation_level": 82,
+  "volume": {
+    "value": 20,
+    "unit": "liters"
+  },
+  "boil_volume": {
+    "value": 25,
+    "unit": "liters"
+  },
+  "method": {
+    "mash_temp": [
+      {
+        "temp": {
+          "value": 65,
+          "unit": "celsius"
+        },
+        "duration": 75
+      }
+    ],
+    "fermentation": {
+      "temp": {
+        "value": 21,
+        "unit": "celsius"
+      }
+    },
+    "twist": ""
```

Comment:
Should be `null`
```suggestion
    "twist": null
```

Reply:
Apologies. All set now. Found a few others from older recipes and fixed them up too,